### ISSUE TITLE
binary extract

### ISSUE THREAD

**Issue by Cicango** [fbiego/watch-face-wearfit#5]
Hi,

Is the binary extract the root of the watch fs getting cloned or is just just watch faces?

**Comment by fbiego** [comment]
For extracting, editing and repacking of watchfaces refer to this forum:
https://4pda.to/forum/index.php?showtopic=1075503&st=20#entry122900599

**Comment by Cicango** [comment]
No I mean clone the whole system partition and dump it for analyzing? Or have I arrived in a watch face repo?

**Comment by fbiego** [comment]
I'm not sure about what you are trying to achieve.

If you are talking about the watch firmware then check here for DT78
https://github.com/fbiego/dt78

**Comment by abidkhancu** [comment]
> I'm not sure about what you are trying to achieve.
> 
> If you are talking about the watch firmware then check here for DT78 https://github.com/fbiego/dt78

I have Chinese Version Watch how can i install english version firmware in it .. my watch working fine with chronOs and Wearfit pro

**Comment by fbiego** [comment]
@abidkhancu have you tried changing the language from the Chronos app? (See image below) If this does not work then it might not be possible

<img width="456" alt="Image" src="https://github.com/user-attachments/assets/320ba705-5d0c-485d-acb4-4b13aab489a8" />

**Comment by abidkhancu** [comment]
> [@abidkhancu](https://github.com/abidkhancu) have you tried changing the language from the Chronos app? (See image below) If this does not work then it might not be possible
> 
> <img alt="Image" width="456" src="https://private-user-images.githubusercontent.com/45796740/407335038-320ba705-5d0c-485d-acb4-4b13aab489a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzgwNzE3MzQsIm5iZiI6MTczODA3MTQzNCwicGF0aCI6Ii80NTc5Njc0MC80MDczMzUwMzgtMzIwYmE3MDUtNWQwYy00ODVkLWFjYjQtNGIxM2FhYjQ4OWE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI4VDEzMzcxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiN2Q2ZGVmMDFlNzU4NGU3OGFjMGJjMDg2M2MzOTEwMjQ4ZDAzYjIzMjNlODJmNWQzMzJmN2VjNTJhZTBkM2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QpeCFE41jfp30Shab0WqPjNGy5BahteJnmJs4xYuXTE">

Yes i have tried it but its not working but i am able to change Dials/Reset watch and do almost everything just not able to change Language do you have any clue or anything for Wearfit Global firmware or anything like that?

**Comment by fbiego** [comment]
Probably they did not include other languages in the firmware

**Comment by fbiego** [comment]
https://www.youtube.com/watch?v=_IKQkJ-mrUc

**Comment by abidkhancu** [comment]
> Probably they did not include other languages in the firmware, it's usually impossible to change the firmware

May be and they mention [https://www.iwhop.com/operatingSystem](https://www.iwhop.com/operatingSystem) here that wearfit os is fully open source so thats why i thought may be Source is Available but i didn't find it anywhere

**Comment by abidkhancu** [comment]
> Probably they did not include other languages in the firmware, it's usually impossible to change the firmware

I have one other question 
i am able to install
https://github.com/fbiego/watch-face-wearfit/tree/main/dials/X5Pro
https://github.com/fbiego/watch-face-wearfit/tree/main/dials/mibro_air
Through your App [Chronos](https://play.google.com/store/apps/details?id=com.fbiego.chronos&hl=en)
but  when i try to Install Online Watch faces the installation stuck it 99% and then failed 

for reference

![Image](https://github.com/user-attachments/assets/902a9ced-d8eb-466b-81b0-8724d57e80b6)
